src/api/Gemfile.lock

```diff
@@ -116,11 +116,11 @@ GEM
     faker (1.7.3)
       i18n (~> 0.5)
     feature (1.4.0)
+    ffi (1.9.17)
```

Comment:
@mdeniz it is an own commit ... did you mean we should put this into another pull request?

Fine for me for that one ....

Btw, buy-in and agreement exists not from all sides, but given that manual work on production data needs to happen also in the affected instance, we want to do it when I am back in 4 weeks.
So this will hang around some time, maybe a good idea to move the small Gemfile.lock in some other pull request then.